enterprise/frontend/src/embedding-sdk/docs/README.md

```diff
@@ -0,0 +1,197 @@
+# Metabase embedding SDK for React
+
+Metabase embedding SDK for React offers a way to integrate Metabase into your application in a more seamless way compared to the current interactive embedding offering.
+
+You need to authentication your user to Metabase using JWT authentication.
+
+## Installation
+
+You can install Metabase embedding SDK for react via npm:
+
+```bash
+npm install @metabase/embedding-sdk-react
+```
+
+or using yarn:
+
+```bash
+yarn add @metabase/embedding-sdk-react
+```
+
+## Usage
+
+Once installed, you need to import `MetabaseProvider` and provide it with a `config` object.
+
+```jsx
+import React from "react";
+import { MetabaseProvider, useQuestionSearch, InteractiveQuestion} from "@metabase/embedding-sdk-react";
+
+// Configuration
+const config = {
+  metabaseInstanceUrl: "https://metabase.example.com", // Required: Your Metabase instance URL
+  authType: "jwt", // Required
+  jwtProviderUri: "https://app.example.com/sso/metabase", // Required: Your endpoint that returns JWT token used to authenticate Metabase. We'll explain more below how to implement this endpoint.
+  font: "Lato", // Optional: you could provide any fonts support by Metabase
+}
+
+export default function App() {
+  const { data: questions, isLoading } = useQuestionSearch();
+
+  if (loading) {
+    return "Loading…"
+  }
+
+  // We just assume the logged in user have access to at least 1 question.
+  const firstQuestion = questions[0]
+
+  return (
+    <MetabaseProvider config={config}>
+      <InteractiveQuestion questionId={firstQuestion.id} />
+    </MetabaseProvider>
+  );
+}
+```
+
+`MetabaseProvider` also supports `pluginsConfig`. Currently we only allow configuring `mapQuestionClickActions`, but we'll support more plugins in next releases.
```

Comment:
For future release, might be good to explain what our plugins system are intended for, and the example use cases for plugins such as mapping click actions. For instance, there is a number of fields below such as `section: "custom", "type": "custom"` that is not immediately obvious what it does, or why I would need it.

Reply:
I hope this makes it a little better 234cc6b0d9caea8d970e11220dfafcb11b538926. But I'd need product people help fleshing this out.